### PR TITLE
Integrate OneDSLoggerWrapper in WebExtensionTelemetry

### PR DESCRIPTION
**Pull request by BidishaMS**
Integrate OneDSLoggerWrapper in WebExtensionTelemetry
Introduce a constant shortNameMappingtoAzureRegion. It will later used in OneDSLogger.

### CODE REVIEW

File: src/web/client/test/integration/AuthenticationProvider.test.ts [hunk 1]
```diff
@@ -67,6 +68,11 @@ describe("Authentication Provider", () => {
             "sendErrorTelemetry"
         );
 
+        const sendErrorTelemetry1 = sinon.spy(
```
Comment (ashishchoudhary001) [inline src/web/client/test/integration/AuthenticationProvider.test.ts:71]: Please rename the const to make it descriptive.

File: src/web/client/extension.ts [hunk 2]
```diff
@@ -89,7 +92,11 @@ export function activate(context: vscode.ExtensionContext): void {
                         );
                     }
                 }
-
+                const geo = queryParamsMap.get('geo')?.toUpperCase();
+                // Authenticated scenario. Pass the geo to OneDSLogger for data boundary
+                if(geo !== undefined && geo !== null && geo !== ''){
```
Comment (ashishchoudhary001) [inline src/web/client/extension.ts:97]: instead of these checks can we replace it by simply checking if geo has a **truthy** value?

**Review by tyaginidhi — APPROVED** [review]

File: src/web/client/test/integration/AuthenticationProvider.test.ts (hunk 1 above)
Comment (BidishaMS) [reply src/web/client/test/integration/AuthenticationProvider.test.ts]: fixed

File: src/web/client/extension.ts (hunk 2 above)
Comment (BidishaMS) [reply src/web/client/extension.ts]: fixed

**Review by ashishchoudhary001 — APPROVED** [review]

**Review by ashishchoudhary001 — APPROVED** [review]

**Review by tehcrashxor — APPROVED** [review]

**Pull request merged**